eloquent/distribution.yaml

```diff
@@ -2203,7 +2203,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/ros/ros_tutorials.git
-      version: eloquent-devel
+      version: eloquent
```

Comment:
I think the `eloquent-devel` branches can remain (and it may make sense to delete the `eloquent` branch. Usually the presence of a -devel branch indicates that the maintainer is sticking with a classic ROS branching strategy of forward-compatible devel branches rather than backward-compatible distro branches which became a thing in ROS 2.

Reply:
Switched this back to eloquent-devel

Reply:
I deleted the created `eloquent` branch in the repo.